### DISCUSSION TITLE
"Killed"  tootctl accounts refresh --all --verbose

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#23358] [Q&A]
When I run the command:

RAILS_ENV=production /home/mastodon/live/bin/tootctl accounts refresh --all --verbose

It starts working for about 5 minutes it says:

Processing 10923947471923773
873/67050

but after 5 minutes it says:

Killed050 |=

and it stops.

What could be the problem?

**Comment by ClearlyClaire** [comment]
Sounds like it may have been killed because of running out of memory (check your kernel logs, via e.g. `dmesg`, for mentions of the OOM killer). I am not sure why it would have run out of memory in the first place but it might be worth investigating.
Out of curiosity, how much memory do you have on the machine you are running this on? Can you try again while monitoring memory usage?

**Reply by ghost** [reply]
I only have 2GB of memory. I checked the dmesg and it says:

oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/user.slice/user-0.slice/session-682.scope,task=ruby,pid=172223,uid=1000
[317936.965365] Out of memory: Killed process 172223 (ruby) total-vm:1847296kB, anon-rss:822444kB, file-rss:1204kB, shmem-rss:0kB, UID:1000 pgtables:3100kB oom_score_adj:0

Looks like you are right!

How much memory do I need? I'm on digital ocean and I can upgrade to 4GB 

I only have like 5 users so i didn't think I need that much ram

**Reply by ClearlyClaire** [reply]
2GB is kind on the lower end of what can handle Mastodon, so I'm not entirely surprised you'd run into issues; but I'd expect you to run into issues generally, not because of `tootctl accounts refresh` specifically. There might be a problem with it.

4GB of RAM for a 5-users server seems reasonable
